src/main/groovy/wooga/gradle/build/unity/ios/internal/utils/PropertyUtils.groovy

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package wooga.gradle.build.unity.ios.internal.utils
+
+import java.util.concurrent.Callable
+
+class PropertyUtils {
+    static String convertToString(Object value) {
```

Comment:
Ah sweet, I remember duplicating it too in Hockey plugin. Would be awesome to have it as a separate package at some point but I remember what you said about unnecessary effort and small packages.

Reply:
Yes. As a funny sidenote on this. My favorite NPM package in the whole world: https://www.npmjs.com/package/isarray

https://github.com/juliangruber/isarray

It has 279 depended packages ;)

The whole implementation is 5 lines long:

```js
var toString = {}.toString;

module.exports = Array.isArray || function (arr) {
  return toString.call(arr) == '[object Array]';
};
```
has 56 lines of test code
and 148 lines of package meta data like `package.json`, `README` (which also includes the 5 lines of implementation), `.gitignore` and so on.

I think this is very extreme form :)